ship_fosdick.gemspec

```diff
@@ -29,7 +29,6 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'spree_core', '~> 2.4'
   spec.add_dependency 'nokogiri'
-  spec.add_dependency 'aws-sdk'
```

Comment:
:boom: and plus the s3 gem requires this I believe so no point in it being in our dependencies.